backend/src/main/kotlin/no/uib/echo/DatabaseHandler.kt

```diff
@@ -173,6 +173,7 @@ class DatabaseHandler(
             "test.mctest@student.uib.no",
             "Test McTest",
             memberships = listOf("webkom"),
+            strike = 0,
```

Comment:
Veldig pirkete, jeg vet, men flertall gir mest mening IMO
```suggestion
            strikes = 0,
```

Reply:
😭